### DISCUSSION TITLE
Laravel app redirect loop via valet server

### DISCUSSION THREAD

**Discussion by lipiniak** [laravel/valet#1457] [Q&A]
Hi, i have a problem with my app running via Valet. When i try to log in to my app i got 302 found error and page is looping to login. 
when i run same app via php artisan serve everything in working properly. Im missing any changes in configirations? I have clean fresh install and only used valet park in my project folder and valet start to start servers.

**Comment by drbyte** [comment]
I think the only "redirect" that Valet would trigger would be from http to https, and if your app has hardcoded URLs to http and valet is trying to serve it via https then you might get endless looping. In such case, remove the hardcoded http:// in your app code.

But, you didn't indicate whether this "loop" is happening with only a certain app/site, or if it's happening on every site. If it happens with multiple sites, are they all Laravel sites, or does it also happen with non-Laravel sites such as a simple index.php file for the site?
